pkg/queue/request_metric.go

```diff
@@ -60,6 +60,11 @@ var (
 		stats.UnitDimensionless)
 )
 
+const (
+	defaultTargetName   = "default"
+	undefinedTargetName = "undefined"
```

Comment:
I'd use uppercase for these to make differences from user defined tags (assuming people can only use lowercase for revision tag).

Reply:
Sounds good!